android/app/src/main/java/org/fossasia/openevent/utils/SharedPreferencesUtil.java

```diff
@@ -0,0 +1,54 @@
+package org.fossasia.openevent.utils;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+
+import org.fossasia.openevent.OpenEventApp;
+
+public class SharedPreferencesUtil {
+    private static SharedPreferences sharedPreferences;
+    private static SharedPreferences.Editor editor;
```

Comment:
SharedPreference holds context as it is dependent on it, but here it is application context so the leak does not happen